<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit</class>
 <widget class="QWidget" name="edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>625</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>625</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="ReceiverLine">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>351</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="MailContent">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>671</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(156, 12, 39);</string>
   </property>
   <property name="text">
    <string>Receiver:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(156, 12, 39);</string>
   </property>
   <property name="text">
    <string>Subject:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubjectLine">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>520</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(156, 12, 39);</string>
   </property>
   <property name="text">
    <string>Content:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Send">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>570</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(193, 101, 110, 170);
border:0;</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Enclosure">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>230</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>625</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(156, 12, 39);
border:0;
background-color: rgb(255, 235, 234);</string>
   </property>
   <property name="text">
    <string>Enclosure</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="BCC">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>85</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>BCC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CC">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>301</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>570</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(193, 101, 110, 170);
border:0;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Write">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/jpg/write.png);
border:0px;</string>
   </property>
   <property name="text">
    <string>        Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Inbox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/jpg/inbox.png);
border:0;</string>
   </property>
   <property name="text">
    <string>        Inbox</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Dustbin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/jpg/dustbin.png);
border:0;</string>
   </property>
   <property name="text">
    <string>        Dustbin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Logout">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>40</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);
border:0;</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Help">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>40</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:0;</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Send_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/jpg/send.png);
border:0;</string>
   </property>
   <property name="text">
    <string>        Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>150</width>
     <height>400</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 182, 181);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Drafts">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/jpg/draft.png);
border:0;</string>
   </property>
   <property name="text">
    <string>        Drafts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Contact">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/jpg/contact.png);
border:0;</string>
   </property>
   <property name="text">
    <string>        Contact</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FengFeng Email</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CC_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>190</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Carbon Copy:(optional)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(193, 101, 110, 170);
border:0;</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123456789@qq.com</string>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>ReceiverLine</zorder>
  <zorder>MailContent</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>SubjectLine</zorder>
  <zorder>label_4</zorder>
  <zorder>Send</zorder>
  <zorder>Enclosure</zorder>
  <zorder>BCC</zorder>
  <zorder>CC</zorder>
  <zorder>Save</zorder>
  <zorder>Write</zorder>
  <zorder>Inbox</zorder>
  <zorder>Dustbin</zorder>
  <zorder>Logout</zorder>
  <zorder>Help</zorder>
  <zorder>Send_2</zorder>
  <zorder>label_7</zorder>
  <zorder>Drafts</zorder>
  <zorder>Contact</zorder>
  <zorder>label</zorder>
  <zorder>CC_2</zorder>
  <zorder>Close</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
